lib/tasks/cognigy/index.js

```diff
@@ -22,7 +23,7 @@ const parseQuickReplies = (obj) => {
 
 const parseBotText = (evt) => {
   const {text, data} = evt;
-  if (text) return text;
+  if (text !== undefined) return String(text);
```

Comment:
what is the purpose of this?  Why are you not calling text.toString() instead of String as a function?

Reply:
It could be that a text is a boolean, instead of a string or a number. In this case we need to explicitly parse it to string.

The explicit undefined check is implemented to also support check for a false value, since otherwise if text === false, we will not run in the if cause even if text is actually defined.